Description: patch pom
 .
 jsqsh (2.1.1-1) unstable; urgency=low
 .
   * Initial release (Closes: #784370)
Author: Scott Wakeling <scott@diskfish.org>
Bug-Debian: https://bugs.debian.org/784370

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- jsqsh-2.1.1.orig/pom.xml
+++ jsqsh-2.1.1/pom.xml
@@ -5,7 +5,7 @@
   <groupId>org.sqsh</groupId>
   <artifactId>jsqsh</artifactId>
   <packaging>jar</packaging>
-  <version>2.1.1-SNAPSHOT</version>
+  <version>2.1.1</version>
   <name>JSqsh</name>
 
   <scm>
@@ -330,9 +330,9 @@
     <dependency>
       <groupId>org.gnu</groupId>
       <artifactId>java-readline</artifactId>
-      <version>0.8.0</version>
+      <version>debian</version>
       <scope>system</scope>
-      <systemPath>${basedir}/lib-nondist/readline-0.8.0.jar</systemPath>
+      <systemPath>/usr/share/java/libreadline-java.jar</systemPath>
     </dependency>
   </dependencies>
 </project>
